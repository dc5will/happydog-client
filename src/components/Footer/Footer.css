.footer {
    display: flex;
    justify-content: center;
    flex-wrap: wrap;
    align-items: center;
    height: 70px;
    background-image: linear-gradient(-90deg, rgb(192, 81, 81), rgb(180, 41, 41));
    color: white;
    padding-top: 10px;
}

.footer p {
    font-size: 14px;
    margin-bottom: 0px;
}

p.footer_brand-container {
    margin: 0px;
}

