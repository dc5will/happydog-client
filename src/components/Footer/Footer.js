import React, { Component } from "react";
import "./Footer.css";

export default class Footer extends Component {
  render() {
    return (
      <footer className="footer">
        <div className="footer_brand-container">
          <p className="footer_author-text">Created by William Wong</p>
          <div className="footer_social-media" />
          <div className="copyright-text">
            <p>Copyright &copy; 2019</p>
          </div>
        </div>
      </footer>
    );
  }
}
