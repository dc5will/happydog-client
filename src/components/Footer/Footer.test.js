import React from "react";
import { shallow } from "enzyme";
import Footer from "./Footer";
import renderer from "react-test-renderer";
import { MemoryRouter } from "react-router-dom";
import Enzyme from "enzyme";
import Adapter from "enzyme-adapter-react-16";

Enzyme.configure({ adapter: new Adapter() });

describe(`LandingPage component`, () => {
  it("renders without crashing", () => {
    shallow(<Footer />);
  });

  it("renders the UI as expected", () => {
    const tree = renderer
      .create(
        <MemoryRouter>
          <Footer />
        </MemoryRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
