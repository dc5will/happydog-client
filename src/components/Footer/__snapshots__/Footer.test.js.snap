// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders the UI as expected 1`] = `
<footer
  className="footer"
>
  <div
    className="footer_brand-container"
  >
    <span
      className="footer_author-text"
    >
      Created by William
    </span>
    <div
      className="footer_social-media"
    />
    <div
      className="copyright-text"
    >
      <p>
        Copyright Â© 2019
      </p>
    </div>
  </div>
</footer>
`;
